packages/react-card/e2e/Card.e2e.tsx

```diff
@@ -0,0 +1,244 @@
+import * as React from 'react';
+import { mount } from '@cypress/react';
+import type {} from '@cypress/react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { webLightTheme } from '@fluentui/react-theme';
+import { Button } from '@fluentui/react-button';
+import { Card, CardFooter, CardHeader } from '@fluentui/react-card';
+import type { CardProps } from '@fluentui/react-card';
+import 'cypress-real-events/support';
```

Comment:
Is this necessary ? I thought it was already registered here: https://github.com/microsoft/fluentui/blob/374b07a1b19e89ce73e7ffe151a40e41775c462f/scripts/cypress/support/index.js#L18

Reply:
That is a mistake, removed 👍🏻